nemo/src/types.rs

```diff
@@ -126,6 +132,23 @@ impl LogicalTypeEnum {
                     }
                 }
             }
+            Self::String => {
+                match gt {
+                    Term::StringLiteral(s) => DataValueT::String(format!("\"{s}\"")),
+                    // TODO: is it correct to support language strings here omitting the language info?
+                    Term::RdfLiteral(RdfLiteral::LanguageString { value, .. }) => {
+                        DataValueT::String(format!("\"{value}\""))
+                    }
```

Comment:
A `LanguageString` is distinct from an `xsd:string` literal (which is what we treat bare string literals as), so I guess dropping the language tag here is not correct. The N-Triples reader currently stores those as `"value"@tag`.

Reply:
Specifically, [RDF 1.1 concepts](https://www.w3.org/TR/rdf11-concepts/#dfn-language-tag) defines `http://www.w3.org/1999/02/22-rdf-syntax-ns#langString` as the type for language tagged strings (but explicitly does not define a datatype for this IRI).